packages/superset-ui-core/test/query/extractQueryFields.test.ts

```diff
@@ -28,7 +29,7 @@ describe('extractQueryFields', () => {
     expect(extractQueryFields({})).toEqual({
       columns: [],
       metrics: [],
-      orderby: [],
+      orderby: undefined,
```

Comment:
Should we test when `metrics` and/or `orderby` are not set too?

Reply:
Some [existing tests](https://github.com/apache-superset/superset-ui/blob/f5c2a8d9ad5c77c06597d082a01e17b1cc009fa0/packages/superset-ui-core/test/query/extractQueryFields.test.ts#L55-L69) already covered this.